pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -23,6 +23,7 @@
 , gtk3Support ? true, gtk2, gtk3, wrapGAppsHook
 , waylandSupport ? true, libxkbcommon
 , gssSupport ? true, kerberos
+, vaapiSupport ? true, libva
```

Comment:
This should probably be version gated. The old esr version doesn't support this I guess?

We would default that to `true` with the next ESR release.

Reply:
Might make sense (e.g. to make it clear which versions support this), but since this only affect `LD_LIBRARY_PATH` it shouldn't be a problem for older versions AFAIK (only provides an additional library that can simply be ignored).

Reply:
This isn't the wrapper. This is in the build expression for firefox.

Reply:
Sorry about the confusion here. I'd never meant to push the source build changes publicly and was hoping I could just get it working. I've decided instead to just keep the changes to the wrapper for this PR. Going to resolve this accordingly.